### PR TITLE
Added reference Key in interface.schema.json file

### PR DESCRIPTION
**Pull request by rekhajp98**
Added reference URL key in the interface.schema.json

### CODE REVIEW

File: Tools/md_generator/generate_md.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def convert_json_to_md():
     print()
     filename = os.path.join(dirname, r"../../*/*Plugin.json")
     flist = glob.glob(os.path.join(filename))
-    jsongenpath ="python ../json_generator/generator_json.py"
+    jsongenpath ="python3 ../json_generator/generator_json.py"
```
Comment (anand-ky) [inline Tools/md_generator/generate_md.py:22]: Checking why we need this change. Won't this break for VMs that use python2? can we separate it out in a different PR?

**Review by anand-ky — CHANGES_REQUESTED** [review]

File: Tools/md_generator/generate_md.py (hunk 1 above)
Comment (RDKDoc) [reply Tools/md_generator/generate_md.py:22]: The minimum requirement to use the JSON generator script is Python 3.5 or higher with pip. It is incompatible with python version 2.

Comment (rekhajp98) [reply Tools/md_generator/generate_md.py:22]: The minimum requirement to use the JSON generator script is Python 3.5 or higher with pip. It is incompatible with python version 2.
As mentioned, I will bring this change in separate PR.
